static/js/components/EmploymentForm.js

```diff
@@ -174,7 +189,7 @@ class EmploymentForm extends ProfileFormFields {
       profile,
     } = this.props;
 
-    let dateFormat = date => moment(date).format("MM[/]YYYY");
+    let dateFormat = date => moment(date).format(DASHBOARD_MONTH_FORMAT);
```

Comment:
same comment about `dateFormat` here as above, maybe this could be pulled out to a util file?